docs/adr/0046-secuploadkeepfiles-directive.md

```
@@ -0,0 +1,89 @@
+
+> "Also, ModSec [requires](https://github.com/owasp-modsecurity/ModSecurity/wiki/Reference-Manual-(v3.x)#secuploadkeepfiles)
+> `SecUploadKeepFiles` directive to work with `SecUploadDir`:
+>
+> > This directive requires the storage directory to be defined (using
+> > `SecUploadDir`).
+>
+> Should we distinguish the dir where we save persisted files from the tmp
+> dir where we save the tmp ones? If so, we should take care of
+> `SecUploadDir` and enforce the same requirement of ModSec"
+> — @M4tteoP ([comment](https://github.com/corazawaf/coraza/pull/1557#issuecomment-4067770063))
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)